gruntfile.js

```diff
@@ -25,21 +25,13 @@ module.exports = function (grunt) {
                 tsconfig: './lib/tsconfig.json'
             },
             dynamicprototest: {
-                tsconfig: './lib/test/tsconfig.json',
-                src: [
-                    './lib/test/Selenium/DynamicProtoTests.ts'
-                ],
-                out: 'lib/test/Selenium/dynamicprototests.js'
+                tsconfig: './lib/test/tsconfig.json'
```

Comment:
These changes also address VSCode pre-checks as the paths within the tsconfig.json are now correct.

The @nevware21/grunt-ts-plugin simplistically just calls ```tsc --project <tsconfig.json>``` while grunt-ts would attempt to create a full command line of the individual files reading some values from tsconfig.json and others from the gruntfile.js